src/main/java/org/cyclonedx/util/LicenseResolver.java

```diff
@@ -66,70 +40,121 @@ private LicenseResolver() {
      * Attempts to resolve the specified license string via SPDX license identifier and expression
      * parsing first. If SPDX resolution is not successful, the method will attempt fuzzy matching.
      * @param licenseString the license string to resolve
-     * @param includeLicenseText specifies is the resolved license will include the entire text of the license
      * @return a LicenseChoice object if resolution was successful, or null if unresolved
      */
-    public static LicenseChoice resolve(String licenseString, boolean includeLicenseText) {
-        try {
-            LicenseChoice licenseChoice = resolveSpdxLicenseString(licenseString, includeLicenseText);
-            if (licenseChoice == null) {
-                licenseChoice = resolveViaAlternativeMapping(licenseString, includeLicenseText);
-            }
-            return licenseChoice;
-        } catch (IOException e) {
-            return null;
-        }
+    public static LicenseChoice resolve(final String licenseString) {
+        return resolve(licenseString, true);
     }
 
     /**
      * Attempts to resolve the specified license string via SPDX license identifier and expression
      * parsing first. If SPDX resolution is not successful, the method will attempt fuzzy matching.
      * @param licenseString the license string to resolve
+     * @param includeLicenseText specifies is the resolved license will include the entire text of the license
      * @return a LicenseChoice object if resolution was successful, or null if unresolved
      */
-    public static LicenseChoice resolve(String licenseString) {
-        return resolve(licenseString, true);
+    public static LicenseChoice resolve(final String licenseString, final boolean includeLicenseText) {
+        final ObjectMapper mapper = new ObjectMapper();
```

Comment:
I think if you wanted to boost performance of using the LicenseResolver class, a constructor could put together that takes an ObjectMapper, and one that doesn't, and the constructor could a) populate mappings, b) license-choice list and then also set the ObjectMapper as long lived on the class. I wouldn't mind doing this at all, but tried to keep the contract as close as possible to what it was, because I didn't want to break upstream consumption!